README.md

```diff
@@ -169,11 +169,7 @@ dc rm -vf $(dc -aq)
 
 > Removes all containers
 
-## Future planes
-
-### Update config
```

Comment:
This section seem to be obsolete, because the command `dc extension edit` already exists.